Source/SwiftLintFramework/Rules/ClosureSpacingRule.swift

```diff
@@ -90,9 +92,12 @@ public struct ClosureSpacingRule: Rule, ConfigurationProviderRule, OptInRule {
         let matchedUpBraces = matchBraces(validBraces(file))
 
         var violationRanges = matchedUpBraces.filter {
-            //removes enclosing brances to just content
+            // removes enclosing brances to just content
             let content = file.contents.substring($0.location + 1, length: $0.length - 2)
-            if content.isEmpty { return false } // case when {} is not a closure
+            if content.isEmpty || content == " " {
```

Comment:
this is kind of ugly, but I didn't want to make multiple spaces valid, only `{}` and `{ }`